### PR TITLE
Clarify quick search by adding a prefix

### LINKED ISSUE #6419: Text entering over the top of commits tree
## Current behaviour
Text displays over the top of commits tree

## Expected behaviour
User shouldn't have possibility to enter text or text should not be displayed.

## Steps to reproduce

1. Click on higher commit on commits tree

2. Start enter text

3. Then you can see text over the top of commits tree

## Did this work in previous version of GitExtensions

I don't now

## Environment

- Git Extensions 3.0.2.5232
- Build 23b6f51905006ccdde8701591df706284b4155dc
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3324.0
- DPI 96dpi (no scaling)

### PR DESCRIPTION
**Pull request by oriash93**
Fixes #6419

## Proposed changes

- Added a string "Searching for: " before the query when performing a quick search, to clarify it's purpose.
- I only changed the label string, so it does not affect the search query nor the logic behind it.

### Before
Untitled3

### After

Untitled2

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by pmiossec** [comment]
Great! But can you please use a 'TranslatedString' so that this string could be translated?

And also, I don't know what others think about it but I think we could improve the change by changing the displayed string when the string is not found (with something like "Searching for (not found):" )

**Comment by oriash93** [comment]
> And also, I don't know what others think about it but I think we could improve the change by changing the displayed string when the string is not found (with something like "Searching for (not found):" )

The text in the label turns red when it's not found, I believe that's enough.

**Comment by pmiossec** [comment]
>The text in the label turns red when it's not found, I believe that's enough.

I made my remark knowing that but because I found it not noticeable enough (a dark red seems to be used). I found it better to have an explicit message for first time users (for me I don't care 😉 ).
But I will let the others decide...

**Comment by RussKie** [comment]
Screenshots?

**Comment by pmiossec** [comment]
>Screenshots?

gitextensions_quick_search

File: GitUI/UserControls/RevisionGrid/QuickSearchProvider.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ internal sealed class QuickSearchProvider
         private readonly RevisionDataGridView _gridView;
         private readonly Timer _quickSearchTimer;
         private readonly IGitRevisionTester _gitRevisionTester;
+        private readonly string _initialString = "Searching for: ";
```
Comment (RussKie) [inline GitUI/UserControls/RevisionGrid/QuickSearchProvider.cs]: This string must be made localisable. 
Please have a look at the `Strings` class, and add this string there.

**Comment by RussKie** [comment]
> but because I found it not noticeable enough (a dark red seems to be used).

How would you make it more noticeable?

**Comment by pmiossec** [comment]
> How would you make it more noticeable?

😄 https://github.com/gitextensions/gitextensions/pull/6450#issuecomment-480616311

Choose your text:  `Searching for (not found):`  or `Text not found:`  (or other)

File: GitUI/UserControls/RevisionGrid/QuickSearchProvider.cs (hunk 1 above)
Comment (oriash93) [reply GitUI/UserControls/RevisionGrid/QuickSearchProvider.cs]: Done

File: GitUI/Translation/English.xlf [hunk 2]
```diff
@@ -9169,6 +9169,10 @@ Details: Is {1} expected {2}</source>
         <source>Comments</source>
         <target />
       </trans-unit>
+      <trans-unit id="_searchingFor.Text">
+        <source>Searching For: </source>
```
Comment (RussKie) [inline GitUI/Translation/English.xlf]:
```suggestion
        <source>Searching for: </source>
```

**Comment by RussKie** [comment]
Thank you

**Pull request merged**